README.md

```diff
@@ -0,0 +1,72 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Book
+
+> 
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+[Live Demo Link]
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+   run the following commands in your terminal
+  - git clone https://github.com/Mhamad-Raad/Book-Store.git
+  - git checkout main
+### Install
+   you can install the following linters into your proejct through node, by these commands:
+  - Light House: npm install -g @lhci/cli@0.7.x
+  - Webhint: npm install --save-dev hint@7.x
+  - Stylelint: npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x    stylelint-csstree-validator@1.x
+  - ESLint: npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+### Deployment
+  you can deploy your project through github pages.
+  - make a repo for your project on github.
+  - go to your project settings.
+  - click on pages.
+  - select the desired branch to deploy.
+  - click save.
+
+## Author
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Mhamad-Raad/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mhamad-raad-446a75227/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/JonahKayizzi)
+
+👤 **Author3**
+
+- GitHub: [@githubhandle](https://github.com/shakerAbuDrais)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used.
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+
```

Comment:
- [ ] Kindly make sure your README file is professional and make and contains true information. For example the title of the project is ` Awesome Book` but the content is not the same